cli/src/commands/upload.ts

```diff
@@ -129,7 +130,6 @@ export class Upload extends BaseCommand {
           }
 
           totalSizeUploaded += asset.fileSize;
```

Comment:
This line needs to be moved up as well. I was just going to do it myself and merge it, but it doesn't look like you've enabled that option.

Reply:
Done